endpoints/openrtb2/sample-requests/invalid-whole/invalid-bidder-params.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This test verifies that when the validation logic is run on a parsed incoming request, the bidder param validation rules are correctly being run since this test shows that an invalid bidder param results in a 400.

Note that we already have `./endpoints/openrtb2/sample-requests/invalid-whole/native-empty.json` which verifies the native validation rules are correctly run since that test shows that an invalid empty native string results in a 400.